src/db/views.ts

```diff
@@ -1,18 +1,26 @@
 import { Table } from './db';
 
-// specify override sql to for creating a view
+// specify override sql for creating complex views for particular tables
 export const overrides: {
   [table in Table]?: string;
 } = {
   [Table.processedTracks]: `
-    select t.*
-    from "${Table.processedTracks}" t
+    select t.* from "${Table.processedTracks}" t
+
     join "${Table.ipfsPins}" p
     on t."lossyAudioIPFSHash" = p.id
     join "${Table.ipfsPins}" p1
     on t."lossyArtworkIPFSHash" = p1.id
+
+    join "${Table.ipfsFiles}" f
+    on t."lossyAudioIPFSHash" = f.cid
+    join "${Table.ipfsFiles}" f1
+    on t."lossyArtworkIPFSHash" = f1.cid
+
     where "lossyArtworkIPFSHash" is not null
     and "lossyAudioIPFSHash" is not null
+    and f."mimeType" is not null
+    and f1."mimeType" is not null
```

Comment:
add extra filters where f."mimeType" is one of audio/video and f1 is image/video